simtools/util/names.py

```diff
@@ -748,3 +755,44 @@ def camera_efficiency_log_file_name(site, telescope_model_name, zenith_angle, la
     name += f"_{label}" if label is not None else ""
     name += ".log"
     return name
+
+
+def get_telescope_type(telescope_name):
+    """
+    Guess telescope type from name. Types are "LST", "MST", "SST", "SCT".
+    Parameters
+    ----------
+    telescope_name: str
+        Telescope name
+    Returns
+    -------
+    str
+        Telescope type.
+    """
+
+    _class, _ = split_telescope_model_name(telescope_name)
+    try:
+        if _class[0:3] in ("LST", "MST", "SST", "SCT"):
+            return _class[0:3]
+
+    except IndexError:
+        pass
+
+    return ""
+
+
+def get_corsika_telescope_data_dict():
```

Comment:
I am sure we read this dict somewhere else, probably in the corsika config module. Perhaps call the function there or write a new function there and call it? Either way, I don't think this should be here in `names.py`

Reply:
It is indeed used in other places. My argument to keep it here is that the function takes parameters from the yml file, i.e. it is related to the names of telescopes and to parameters that are not going to change as their sizes and observation level. If you prefer I can move it to util.general ?

Reply:
Perhaps I misunderstand and a quick chat would explain it, but the way I see it, we should read this file once for each "run" of gammasim-tools. Then you have the entire dictionary "in memory" and each module that needs it retrieves it and extracts the info it needs from it. This is exactly what we do with the model parameters and it works great. 

That's what I suggest to do here, read this file in `corsika_config` (or somewhere there) and call that function here. It's better to keep it in a class like `corsika_config` rather than in `util.general` cause then you can make sure you read the file only once per instance of `corsika_config` (which is essentially only once per "run" of gammasim-tools) rather than every time you call the namespace `util.general`.